proxy/contracts/predeployed/TokenManager.sol

```diff
@@ -44,7 +44,14 @@ interface ILockAndDataTM {
 // This contract runs on schains and accepts messages from main net creates ETH clones.
 // When the user exits, it burns them
 
-
+/**
+ * @title Token Manager
+ * @dev Runs on SKALE Chains, accepts messages from mainnet, and instructs
+ * TokenFactory to create clones. TokenManager manages ETH, ERC20 and ERC721
```

Comment:
I think is a duplication of the next sentense

Reply:
Clarified.